packages/cms/pages/auth/register/register.tsx

```diff
@@ -1,23 +1,82 @@
-import React from 'react'
+import React, { FormEvent } from 'react'
 import styled from 'styled-components'
 import { AuthLayout } from '../../components/auth/layout'
 
 import { EuiText } from '@tensei/eui/lib/components/text'
 import { EuiLink } from '@tensei/eui/lib/components/link'
 import { EuiTitle } from '@tensei/eui/lib/components/title'
-import { EuiFormRow } from '@tensei/eui/lib/components/form'
+import { EuiForm, EuiFormRow } from '@tensei/eui/lib/components/form'
 import { EuiButton } from '@tensei/eui/lib/components/button'
 import { EuiSpacer } from '@tensei/eui/lib/components/spacer'
 import { EuiFlexGroup } from '@tensei/eui/lib/components/flex'
 import { EuiFieldText } from '@tensei/eui/lib/components/form/field_text'
 import { EuiFieldPassword } from '@tensei/eui/lib/components/form/field_password'
 import { EuiFlexItem } from '@tensei/eui/lib/components/flex/flex_item'
+import { useState } from 'react'
+import { RegisterCredentials, useAuthStore } from '../../../store/auth'
 
 const H3 = styled.h3`
   text-align: center;
 `
 
 export const Register: React.FunctionComponent = () => {
+  const { register } = useAuthStore()
+
+  const [user, setUser] = useState<RegisterCredentials>({
+    firstName: '',
+    lastName: '',
+    email: '',
+    password: ''
+  })
+
+  const [errors, setErrors] = useState<{
+    firstName: String[]
```

Comment:
Let's use `string` instead of `String`. Even though typescript allows for both, one is a constructor and another is a type. Let's use the type.